scripts/lint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+echo "[LINT] Checking JavaScript"
+eslint -c .eslintrc.prod.json src
+
+echo "[LINT] Checking Stylesheets"
+stylelint "src/**/*.{scss,css}"
```

Comment:
Only checking `*.css` and `*.scss` files, because the LESS source files will automatically get linted as we transition to SCSS.

Reply:
Later on, we'll update this to include `docs/**/*.{scss,css}` too.

Reply:
Do you want to lint always all files, or only files modified?

Reply:
Always lint all files. Linting changed files is an optimization that is negligible at this point, given that it doesn't take long to run linting and we're going to be updating the pipeline logic, anyway.